components/indexer.py

```diff
@@ -100,7 +86,11 @@ def has_cargo_in_tunnel(self) -> bool:
 
     @feedback
     def last_cargo_was_opposition(self) -> bool:
-        return self.last_colour.is_opposition()
+        if wpilib.DriverStation.getAlliance() is wpilib.DriverStation.Alliance.kBlue:
+            return self.red_total > self.blue_total
+        elif wpilib.DriverStation.getAlliance() is wpilib.DriverStation.Alliance.kRed:
```

Comment:
pybind11 enums are weird :(
```suggestion
        if wpilib.DriverStation.getAlliance() == wpilib.DriverStation.Alliance.kBlue:
            return self.red_total > self.blue_total
        elif wpilib.DriverStation.getAlliance() == wpilib.DriverStation.Alliance.kRed:
```
although we could just do away with this altogether?